src/utility/Ex4.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import scipy.stats as stats
+import json
+jsonfile=open('data\input.json','r')
+readfile=jsonfile.read()
+data=json.loads(readfile)
+ward1=data["wardDistribution"].get("distribution")
+ward=[]
+for each in ward1:
+    ward.append(each)
+ward.pop()
+#doc du lieu triple tu Pedestrian.json
+triple=0
+n=data["numOfAgents"].get("value")
+jsonfile2=open('data\Pedestrian.json','r')
+readfile2=jsonfile2.read()
+data2=json.loads(readfile2)
+for each in data2:
+    if each["type"]=="Personel":
+        triple+=1
+single=n-triple
+print(triple)
+#sinh du lieu
+max_value=n-len(ward)*3
```

Comment:
I don't think this `max_value` would be viable. What if `numOfAgents` is a really big number (say around 40k) and `wardCount` isn't big enough?

Reply:
This max_value should greater than numOfAgents/len(ward).

Reply:
avg=n*0.1(since len(ward)=10). So i suppose max_value is about n*0.3. Test with n=10000 ~40000 and triple=n/3 or n/4 or n/5 works good

Reply:
I think `max_value = 0.3 * n` seems to work just fine. Unless you have a better option, we can just do it like that and call it a day! Great job! :D